docs/src/main/sphinx/ext/download.py

```diff
@@ -17,80 +17,78 @@
 # noinspection PyUnresolvedReferences
 from sphinx.errors import SphinxError
 
-GROUP_ID = 'io.trino'
 ARTIFACTS = {
-    'server': ('trino-server', 'tar.gz', None),
-    'server-core': ('trino-server-core', 'tar.gz', None),
-    'cli': ('trino-cli', 'jar', 'executable'),
-    'jdbc': ('trino-jdbc', 'jar', None),
+    'server': ('trino-server', 'tar.gz'),
+    'server-core': ('trino-server-core', 'tar.gz'),
+    'cli': ('trino-cli', None),
```

Comment:
Is this going to work? The filename should be like https://repo1.maven.org/maven2/io/trino/trino-cli/476/trino-cli-476-executable.jar

Reply:
No, it can be anything we want. We’ll publish those artifacts to github, so we’ll change the name. No need for the “-executable” qualifier

Reply:
Ok .. so the filename will be just trino-cli-476.jar ? If so I have to adjust my PR for the website .. which is fine by my .. just might be confusing for readers of the book or older docs for what they are used to.  Just let me know so I can adjust the other PR

Reply:
No, it will be just “trino-cli-476”. It’s just a binary executable

Reply:
Uff .. I think thats actually a bad idea. It is not a pure executable .. it still requires a JVM on the path .. and with the .jar extension it can be recognized and run .. without thats harder.

Reply:
Kind of feels like a unrelated change that we should not pull in unless we have to.

Reply:
Wdyt @wendigo @electrum ? I know the file works with the name without extension if Java is on the path, but if not it might be a bad experience. Have any of you played with that in the past?

Reply:
The extension doesn’t actually do anything. The file behaves like a script with a shebang line.